multidim-interop/src/generator.ts

```diff
@@ -111,8 +110,6 @@ export async function buildTestSpecs(versions: Array<Version>): Promise<Array<Co
                 dialerID: test.id1,
                 listenerID: test.id2,
                 transport: test.transport,
-                muxer: "quic",
-                security: "quic",
```

Comment:
Can we do this in a separate PR?

Reply:
I thought about it, but there's a lot of overhead for doing all these things in separate PRs. I reorganized the commits so that they can be reviewed commit by commit.